src/services/translation_service/translation_service.cpp

```diff
@@ -308,6 +308,14 @@ namespace big
 				case 12: preferred_lang = "zh_CN"; break;
 				}
 
+				if (game_lang == 12 || game_lang == 9)
+				{
+					// Tweaks to make it easier for people playing in the China region
+					g.session_browser.filter_multiplexed_sessions = true;
+					g.reactions.chat_spam.block_joins             = true;
+					g.reactions.chat_spam.block_join_reason       = 27;

```

Comment:
What is join_reason 27? Is this those god-awful annoying spambots? Shouldn't we do this for all regions?

Reply:
The usage here of "27" here like that is truly awful, related to https://github.com/YimMenu/YimMenu/issues/3289, non blocking cause I'll get around it at some point